exercises/sum-of-multiples/sum_of_multiples.example

```diff
@@ -0,0 +1,14 @@
+factored([Curr|Factors], Num) :-
+    0 is mod(Num, Curr), !;
+    factored(Factors, Num).
+
+sum_of_multiples(_, 0, 0) :- !.
```

Comment:
Does this work as `sum_of_multiples(_, 0, 0).` or does that throw a warning in test?